app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_split = command.split()
+    command_to_read, command_to_write = command_split
```

Comment:
You don't need to separate it between two lines, wi be better to use `command, old_file, new_file = command.split()`

Reply:
Don't forget that you also must have "cp" in your command